schemas/json/layout/layout.schema.v1.json

```diff
@@ -247,6 +248,16 @@
             "description": "String value indicating which layout page the referenced component is defined on."
           }
         }
+      },
+      "attachmentListComponent": {
+        "properties": {
+          "dataTypeIds": {
+            "type": "array",
+            "title": "Data type IDs",
+            "description": "List of data type IDs for the attachment list to show.",
+            "examples": [["SomeDataType", "SomeOtherDataType"]]
```

Comment:
Is the double array intended?

Reply:
Missing type definition for array items?
```json
"items": {
      "type": "string"
 }
```

Reply:
`"examples"` an array of examples, thus the double array :)  
![image](https://user-images.githubusercontent.com/1636323/105166967-6c584900-5b18-11eb-8919-600ec8c349ef.png)

Fixed missing type definition for array items